wgengine/netstack/netstack.go

```diff
@@ -62,7 +62,7 @@ func Impl(logf logger.Logf, tundev *tstun.TUN, e wgengine.Engine, mc *magicsock.
 		log.Fatal(err)
 	}
 
-	ipstack.AddAddress(nicID, ipv4.ProtocolNumber, tcpip.Address(net.ParseIP("100.96.188.101").To4()))
+	ipstack.AddAddress(nicID, ipv4.ProtocolNumber, tcpip.Address(net.ParseIP("100.110.9.22").To4()))
```

Comment:
revert. unrelated.

but you could fix this separately too before your big change.

Reply:
Yeah, I'll revert this and make a more permanent fix after this PR